osl/interface_linux.go

```diff
@@ -335,6 +350,18 @@ func setInterfaceIP(iface netlink.Link, i *nwIface) error {
 	return netlink.AddrAdd(iface, ipAddr)
 }
 
+func setInterfaceIPAliases(iface netlink.Link, i *nwIface) error {
+	if i.IPAliases() == nil {
```

Comment:
Besides `IPAliases()` never returns nil, you can remove this check, the `for range` loop correctly handles empty and nil list.